core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -101,27 +101,29 @@ describe('Overview', () => {
             browser.wait(EC.visibilityOf(jazzServices_po.getAssetHeader()), timeOutHigh);
             browser.sleep(4000);
             browser.wait(EC.elementToBeClickable(jazzServices_po.getServiceFromAsset()), timeOutHigh);
-            jazzServices_po.getServiceFromAsset().click();
       });
 
-      xit('Verify Logs' ,  () => {
+      it('Verify Logs' ,  () => {
             browser.wait(EC.visibilityOf(jazzServices_po.getLogs()), timeOutHigh);
             jazzServices_po.getLogs().click();
             browser.wait(EC.visibilityOf(jazzServices_po.getFilterIcon()), timeOutHigh);
             browser.driver.switchTo().activeElement();
             jazzServices_po.getFilterIcon().click();
             jazzServices_po.getDropDown().click();
-            //jazzServices_po.getDay().click();
-            browser.sleep(4000);
+            jazzServices_po.getDay().click();
+            browser.sleep(2000);
             jazzServices_po.getDropDown().click();
             jazzServices_po.getWeek().click();
-            browser.sleep(4000);
+            browser.sleep(2000);
+		expect(jazzServices_po.getWeekVerify().getText()).toEqual('WEEK');
             jazzServices_po.getDropDown().click();
             jazzServices_po.getMonth().click();
-            browser.sleep(4000);
+            browser.sleep(2000);
+		expect(jazzServices_po.getMonthVerify().getText()).toEqual('MONTH');
```

Comment:
i think there is 'DAY' as well?

Reply:
Sid,Day is coming by default,so i think no need for verification there ,if the following "Week" verification fails (for any reason) the test will not pass.so Week,Mont,Year is enough since day is coming by default